### PR TITLE
Add populate methods + delete function + accept/reject offers

### LINKED ISSUE #17: No error handling when task is deleted
TODO: add an error handling to display Task has been deleted or doesnt exist to page view of task

### PR DESCRIPTION
**Pull request by karrui**
Follow-up of #15 
Fixed #17 and #21

* Add `New task` button on header of the page.
* Add functions to populate sample data into the database.
* Add `Delete` button in `Edit` task.
* Fix DatePicker bug.
* Add a SQL query to drop all existing functions, to avoid duplicates.
* Add pages for displaying offers by `status_offer` by `open`, `rejected` and `accepted`.
* Change data type of `price`, in all tables, from `MONEY` to `NUMERIC (6, 2)`.
* Enable accepting/rejecting offers.

### CODE REVIEW

**Comment by karrui** [comment]
Also implements and fixes #17

**Comment by karrui** [comment]
https://github.com/karrui/taskrr/pull/18/commits/2fce7d0bcd027f3eb8ac10f7652d64d516b98f8b implements and fixes #17

File: db/queries/insert_queries.js [hunk 1]
```diff
@@ -12,11 +12,17 @@ _location TEXT,
...
+// TODO: Functions not working yet, made an insert
+// exports.ONE_TASK = `
+//     SELECT
+//         insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)
+//     ;
+// `
 exports.ONE_TASK = `
-    SELECT
-        insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)
+    INSERT INTO task (title, description, category_id, location, requester, start_dt, end_dt, price)
```
Comment (karrui) [inline db/queries/insert_queries.js]: @snguyenthanh This was buggy due to functions not working previously so I did this, change it back if you want, haha

Comment (snguyenthanh) [reply db/queries/insert_queries.js]: It was not the function's bug but duplicate functions by keeping on creating functions when the app starts.
The bug has been resolved by dropping all functions before creating any, when the app starts.
I will revert back to `SELECT insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)`.

**Comment by snguyenthanh** [comment]
How the accept/reject offer Functions work is described in #15 

@karrui Can you change the how the offers in page will work according to the functions ?
* When an offer is rejected, remain the same as now.
* But when an offer is accepted, show all other offers as rejected.

**Comment by snguyenthanh** [comment]
@karrui I have updated the `description` section of the PR.

**Comment by snguyenthanh** [comment]
@karrui @TTTaus Please have a final check for all functions, before merging.

**Comment by karrui** [comment]
@snguyenthanh Do a final check, fixed the merge conflict

**Comment by snguyenthanh** [comment]
All checks passed and no bugs found.

**Pull request merged**